src/bindings/python/src/openvino/runtime/opset15/ops.py

```diff
@@ -39,3 +39,45 @@ def scatter_nd_update(
     if reduction:
         attributes["reduction"] = reduction
     return _get_node_factory_opset15().create("ScatterNDUpdate", inputs, attributes)
+
+
+@nameable_op
+def col2im(
+    data: NodeInput,
+    output_size: NodeInput,
+    kernel_size: NodeInput,
+    strides: Optional[List[int]] = None,
+    dilations: Optional[List[int]] = None,
+    pads_begin: Optional[List[int]] = None,
+    pads_end: Optional[List[int]] = None,
+    name: Optional[str] = None,
+) -> Node:
+    """Perform data movement operation which combines sliding blocks into an image tensor.
+
+    :param  data:                The node providing input data.
+    :param  output_size:         Shape of the spatial dimensions of the output image.
+    :param  kernel_size:         Size of the sliding blocks.
+    :param  strides:             Stride on the sliding blocks in the input spatial dimensions. Defaults to [1, 1].
+    :param  dilations:           The dilation of filter elements (distance between elements). Defaults to [1, 1].
+    :param  pads_begin:          The number of pixels to add at the beginning along each axis. Defaults to [0, 0].
+    :param  pads_end:            The number of pixels to add at the end along each axis. Defaults to [0, 0].
+    :param  name:                The optional name for the created output node.
+
+    :return:   The new node performing Col2Im operation.
+    """
+    for param in [strides, dilations]:
+        if param is None:
+            param = [1, 1]
+    for param in [pads_begin, pads_end]:
+        if param is None:
+            param = [0, 0]
```

Comment:
Can you test these `None` cases somehow?

Reply:
They are tested in the first test case: `((3, 4), [2, 2], [1, 1], [3, 2, 2], []),`. The last empty list is `params`, which are given into the op constructor as `*params`.